lib/ProductOpener/CRM.pm

```diff
@@ -0,0 +1,257 @@
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2023 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+=encoding UTF-8
+
+=head1 NAME
+
+ProductOpener::CRM - manages integration with the Odoo CRM
+
+=head1 SYNOPSIS
+
+C<ProductOpener::CRM> contains functions to interact with the Odoo CRM
+
+
+=head1 DESCRIPTION
+
+[..]
+
+=cut
+
+package ProductOpener::CRM;
+
+use ProductOpener::PerlStandards;
+use Exporter qw< import >;
+
+BEGIN {
+	use vars qw(@ISA @EXPORT_OK %EXPORT_TAGS);
+	@EXPORT_OK = qw(
+		&create_company
+		&create_contact
+		&add_contact_to_company
+		&create_opportunity
+		&test
+	);
+	%EXPORT_TAGS = (all => [@EXPORT_OK]);
+
+}
+use vars @EXPORT_OK;
+
+use ProductOpener::Tags qw(%country_codes_reverse);
+use XML::RPC;
+use Log::Any qw($log);
+use Data::Dumper;
+use feature 'say';
```

Comment:
`Data::Dumper` and `say` aren't actually used, right? If they are, it _might_ be better to use a proper `$log` method call instead